Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfully.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void authorization transaction](01_ManageAuthorizations.md#void-authorization-transaction).
+    
+2. Click on the authorization transaction to see the details.   
+    The sub-tabs of the transaction are opened.   
+     
+    *Payments > Authorizations > Tab LIST > Select Authorization transaction*  
+
+    ![Authorization attributes and logs](../../Assets/Screenshots/Payments/Authorizations/CheckAttributes.png "[Authorization attributes and logs]")
+
+3. Check the attributes and logs of the transaction. For detailed information, see the following:
+   - [Authorizations &ndash; Attributes](../UserInterface/01a_ListAuthorizations.md#authorizations-–-attributes)
+   - [Authorizations &ndash; Logs](../UserInterface/01a_ListAuthorizations.md#authorizations-–-logs)
+4. Return to the LIST and decide how to proceed with the transaction.
+
+
+## Capture authorization transaction
+
+After you have checked an authorization transaction that could not be processed by the workflow, you can capture the transaction manually, so that the order process can be continued.
+
+#### Prerequisites
+
+- The status of an authorization transaction is **not** "Success". In this case, further processing has already been continued.<!---Stimmt das oder gibt es noch mehr?-->   
+
+
+#### Procedure
+*Payments > Authorizations > Tab LIST*
+
+![Capture authorization transaction](../../Assets/Screenshots/Payments/Authorizations/ChangeAuthorization.png "[Capture authorization transaction]")   
+
+1. Check the authorization transaction you want to capture, see [Check authorization transaction](./01_ManageAuthorization.md#check-authorization-transaction).
+2. Select the authorization transaction you want to capture by clicking the checkbox on the left.   
+    The editing toolbar is displayed.
+3. Click the [Capture] button.    
+   A confirmation message is displayed. The status of the authorization transaction has changed to "Success".   
+   You can now follow up this case under the menu entry *Payments and captures*. <!---Stefan ist das richtig?--> 
+   
+  
+
+## Cancel authorization transaction
+
+After you have checked an authorization transaction that could not be processed by the workflow, you can cancel the transaction manually, so that the order is cancelled both at Actindo and at the payment service provider. Actindo will then automatically inform the payment service provider about the cancellation. The customer will get his or her money back.
```

Comment:
Wenn ich es richtig verstanden habe, wurde das Geld noch nicht abgebucht, sondern nur reserviert, daher "The customer will get his or her money back" stimmt nicht ganz hier.

Reply:
Done